ReactWindows/ReactNative.Net46/Tracing/Tracer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace ReactNative.Tracing
+{
+    class NullDisposable : IDisposable
+    {
+        public void Dispose()
+        {
+
+        }
+    }
+
+    public class NullLoggingActivityBuilder
```

Comment:
I can add the documentation. Removing `public` causes `insufficient privileges` errors on compile.